common/src/main/java/com/tencent/rss/common/config/ConfigUtils.java

```diff
@@ -98,6 +98,8 @@ static Long convertToLong(Object o) {
       return ((Integer) o).longValue();
     }
     if (UnitConverter.isByteString(o.toString())) {
+      // Use long type as size type as much as possible,
+      // so UnitConverter only support long type
```

Comment:
add the comment `intType only support the size that less than 4GB`

Reply:
fixed